src/content/ccip/supported-networks/v1_2_0/mainnet.mdx

```diff
@@ -0,0 +1,37 @@
+---
+section: ccip
+date: Last Modified
+title: "CCIP Supported Networks Mainnet - v1.2.0"
+---
+
+import { Aside } from "@components"
+import Main from "@features/ccip/components/supported-networks/Main.astro"
+import { Environment, Version } from "@config/data/ccip"
+
+<Aside type="note" title="Mainnet Access">
+  [Apply for Mainnet Early Access](https://chainlinkcommunity.typeform.com/ccip-form?#ref_id=ccip_docs) to start
+  building on mainnet.
+</Aside>
+
+<Aside type="note" title="Early Access">
```

Comment:
I noticed you removed the Early Access notice for 1.2.0 testnet. Does this notice still apply to 1.2.0 Mainnet?